src/runtime/graph_executor/debug/graph_executor_debug.cc

```diff
@@ -140,6 +140,11 @@ class GraphExecutorDebug : public GraphExecutor {
       return 0;
     }
 
+    if (nodes_[index].param.func_name == "__nop") {
+      LOG_INFO << "Skipping __nop function";
```

Comment:
```suggestion
      LOG(DEBUG) << "Skipping __nop function";
```

Reply:
Thanks @tkonolige. when I tried LOG(DEBUG), I got an error stating that "‘LOG_DEBUG’ was not declared in this scope". Not exactly sure why, maybe because we are already in debug mode when using graph_executor_debug?